_pages/en_US/faq.md

```diff
@@ -6,30 +6,49 @@ redirect_from:
 {% include toc title="Questions" %}
 
 ### What does a white screen in certain homebrew mean?
-That implies an SD card error. Please make sure you have [checked your SD card for errors](sd-prep).
+That implies an SD card error. Please make sure you have [checked your SD card for errors](sd-card-setup).
 
-### How do I launch retail ROMs?
-Retail ROMs are not able to be launched through Unlaunch alone, as Retail ROMs was made to be used by Slot-1. In order to run them from your SD card, you'll need to use nds-bootstrap, an application that can redirect Slot-1 reads to the SD card. It knows what to launch based on the strings set in the configuration file. While manually setting up the configuration file is possible, it's tedious and isn't user friendly. Depending on how you'd like to access your ROMS, there are easier ways.
-- hiyaCFW users can create [forwarders](forwarder) for the DSi System Menu, but it is limited to 40 title and involves more steps than drag and drop.
-- TWiLight Menu++ can navigate your SD card to discover and launch your ROM files. This has the benefit of no 40 title limit, drop and play (no conversion required), and no Unlaunch requirement.
+### What functionality would I lose by modding my system?
 
-Keep in mind that we would prefer if you [dumped these carts using GodMode9i](dump-cart). We do not support software piracy, and please do not go asking us how to get you free games.
+- If you decide to install Unlaunch, you will not lose any system functionality
+- If you go with a memory-pit only setup, you will be unable to save photos to the SD card via the Nintendo DSi Camera Application while the exploit is already active. Either switch to an [alternative exploit](alternative-exploits) or [install Unlaunch](/installing-unlaunch).
+
+This is because we overwrite the metadata file (`pit.bin`) with the Memory Pit exploit.
+
+### How do I launch Nintendo DS cartridge dumps?
```

Comment:
Maybe "play" instead of "launch" would be better? Launch isn't wrong or anything, but in my head I think play sounds better since its about games